app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
@gaganawhad @chrismaddocks something about all of these suggestions just feels wrong. EDD ftw.

1.) I think the naming collision between the `host` parameter of the `#url` method and the `#host` attribute of the `Wakes::Location` model is a code smell. What we probably want to do is call the parameter something like `override_host`. If we could do that, then the code here would be much easier to understand: `host = override_host || self.host || ENV['DEFAULT_HOST']`. I would prefer the explicitness we gain from this over an attempt to match things to the route helper API. This method should _almost never_ need to take any parameters.

2.) That then pushes us to look at the situations where we'd want to actually override the host. For redirects, you're going to be going off of paths unless a host is explicitly specified. You shouldn't need to override there. So it's only for metrics. And it seems like the use case for metrics is in scripts and console. In that case, you're probably going to want to override across the board, not just one by one. That is, you're not going to want a different host for every url, but the same host. So I wonder if there's some sort of global config that would be better suited, whether it's the `ENV` var or something that gets initially set to that var.